src/transforms/wrapperize.jl

```diff
@@ -42,6 +42,40 @@ function get_wrapper_rules(alg, depth, ctx)
                 end
             end
         end),
+        (@rule call(<, ~i::isindex, ~j::isindex) => begin
+            if depth(i) < depth(j)
+                access(LoTriMask(), reader(), call(-, i, 1), j)
+            elseif depth(i) > depth(j)
+                access(UpTriMask(), reader(), call(+, i, 1), j)
+            end
+        end),
```

Comment:
Could we refactor each of these rules into two cases:
```
        (@rule call(<, ~i, ~j::isindex) => begin
            if depth(i) < depth(j)
                access(UpTriMask(), reader(), j, call(-, i, 1))
            end
        ),
        @rule call(<, ~i::isindex, ~j) => begin
            if depth(i) > depth(j)
                access(LoTriMask(), reader(), i, call(+, j, 1))
            end
        ),
 ```
 This code is different in a few ways:
 1. Only the inner (fast) index needs to be a vanilla index (as opposed to `i + 1`, etc.).
 2. This code flips the index order in order to always emit a column major access.